api/Helpers.py

```diff
@@ -13,10 +13,10 @@ def create_user(username, password):
             users.append(user)
 
 
-def login_user(username):
+def login_user(username, password):
     """function that allows a known user login """
     for user in users:
-        if user.username == user['username']:
+        if user.username == user['username'] and user.password == user['password']:
```

Comment:
TO compare verify the password, please make use of the         "check_password_hash() function"

Reply:
Thanks bro...